broker-daemon/broker-rpc/wallet-service/unlock-wallet.js

```diff
@@ -0,0 +1,30 @@
+const { PublicError } = require('grpc-methods')
+
+/**
+ * Unlock an engine's wallet
+ *
+ * @param {GrpcUnaryMethod~request} request - request object
+ * @param {Logger} request.logger
+ * @param {Object<String>} request.params
+ * @param {String} request.params.symbol - currency symbol of the wallet e.g. `BTC`
+ * @param {String} request.params.password - password for the specified engine's wallet
+ * @param {Map<Engine>} request.engines
+ * @param {Object} responses
+ * @param {Function} responses.EmptyResponse
+ * @return {EmptyResponse}
+ */
+async function unlockWallet ({ logger, params, engines }, { EmptyResponse }) {
+  const { symbol, password } = params
+  const engine = engines.get(symbol)
+
+  if (!engine) {
+    logger.error(`Could not find engine: ${symbol}`)
+    throw new PublicError(`Unable to create wallet for engine: ${symbol}`)
```

Comment:
`create wallet` is kind of strange here, why not `Unable to unlock wallet`? Also, do you want to provide more feedback to the user that it is a potentially invalid symbol?

Reply:
Im going to change this message to something generic instead of `unlock wallet` so that it is consistent across the calls.